SandboxiePlus/SandMan/sandman_vi.ts

```diff
@@ -0,0 +1,5904 @@
+        <translation type="unfinished">Bảo vệ các tiến trình trong Sandbox này khỏi bị truy cập bởi các quy trình máy chủ lưu trữ không có Sandbox được chỉ định.</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3472"/>
+        <location filename="Forms/OptionsWindow.ui" line="3536"/>
+        <source>Process</source>
+        <translation type="unfinished">Tiến trình</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3512"/>
+        <source>Block also read access to processes in this sandbox</source>
+        <translation type="unfinished">Chặn cũng đọc quyền truy cập vào các tiến trình trong Sandbox này</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3562"/>
+        <source>Users</source>
+        <translation>Người dùng</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3580"/>
+        <source>Restrict Resource Access monitor to administrators only</source>
+        <translation>Giới hạn giám sát Quyền truy cập tài nguyên chỉ dành cho quản trị viên</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3587"/>
+        <source>Add User</source>
+        <translation>Thêm người dùng</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3617"/>
+        <source>Add user accounts and user groups to the list below to limit use of the sandbox to only those accounts.  If the list is empty, the sandbox can be used by all user accounts.
+
+Note:  Forced Programs and Force Folders settings for a sandbox do not apply to user accounts which cannot use the sandbox.</source>
+        <translation>Thêm tài khoản người dùng và nhóm người dùng vào danh sách bên dưới để giới hạn việc sử dụng Sandboxchỉ cho những tài khoản đó. Nếu danh sách trống, Sandbox có thể được sử dụng bởi tất cả các tài khoản người dùng.
+
+Ghi chú:  Cài đặt Buộc chương trình và Thư mục bắt buộc cho Sandbox không áp dụng cho các tài khoản người dùng không thể sử dụng Sandbox. </translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3662"/>
+        <source>Add Option</source>
+        <translation type="unfinished">Thêm tùy chọn</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3676"/>
+        <source>Here you can configure advanced per process options to improve compatibility and/or customize sandboxing behavior.</source>
+        <oldsource>Here you can configure advanced per process options to improve compatibility and/or customize sand boxing behavior.</oldsource>
+        <translation type="unfinished">Tại đây, bạn có thể định cấu hình các tùy chọn nâng cao cho mỗi quy trình để cải thiện khả năng tương thích và / hoặc tùy chỉnh hành vi Sandbox. </translation>
```

Comment:
```suggestion
        <translation type="unfinished">Tại đây, bạn có thể định cấu hình các tùy chọn nâng cao cho mỗi quy trình để cải thiện khả năng tương thích và / hoặc tùy chỉnh hành vi Sandbox.</translation>
```

The extra space is not needed.